src/Microsoft.AspNetCore.Mvc.Testing/Internal/TestCulture.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.Testing.Internal
+{
+    public class TestCulture
+    {
+        public string Culture { get; set; }
```

Comment:
It's an implementation detail (pubternal), the difference is minimal, so I don't think it adds any value. Let me know if you feel strongly about this and I'll change it